File: core/src/org/rexcellentgames/burningknight/debug/RoomDebugCommand.java
package org.rexcellentgames.burningknight.debug;

import org.rexcellentgames.burningknight.entity.level.Level;

public class RoomDebugCommand extends ConsoleCommand {
	{
		name = "/room";
		shortName = "/r";
	}

	@Override
	public void run(Console console, String[] args) {
		Level.RENDER_ROOM_DEBUG = !Level.RENDER_ROOM_DEBUG;
	}
}

File: core/src/org/rexcellentgames/burningknight/debug/RoomDebugCommand.kt
package org.rexcellentgames.burningknight.debug

import org.rexcellentgames.burningknight.entity.level.Level

class RoomDebugCommand : ConsoleCommand("/room", "/r", "Some room debug") {
	override fun run(console: Console, args: Array<String>) {
		Level.RENDER_ROOM_DEBUG = !Level.RENDER_ROOM_DEBUG
	}
}